database/src/main/scala/cromwell/database/slick/BackendKVStoreSlickDatabase.scala

```diff
@@ -11,12 +11,20 @@ trait BackendKVStoreSlickDatabase extends BackendKVStoreSqlDatabase {
   import dataAccess.driver.api._
 
 
-  override def addBackendKVStoreEntry(entry: BackendKVStoreEntry)(implicit ec: ExecutionContext): Future[Unit] = {
+  override def addBackendJobKeyValuePair(workflowID: String, callFqn: String, callIndex: Int, callAttempt: Int, storeKey: String, storeValue: String)(implicit ec: ExecutionContext): Future[Unit] = {
+
+    val entry = BackendKVStoreEntry(workflowID.toString,
+                              callFqn,
+                              callIndex,
+                              callAttempt,
+                              storeKey,
+                              storeValue)
+
     val action = dataAccess.backendKVStoreAutoInc += entry
     runTransaction(action) map { _ => () }
   }
 
-  override def queryBackendKVStoreValueByStoreKey(workflowUid: String, callFqn: String, callIndex: Int, callAttempt: Int, jobKey: String)(implicit ec: ExecutionContext): Future[Option[String]] = {
+  override def queryBackendJobValueByJobKey(workflowUid: String, callFqn: String, callIndex: Int, callAttempt: Int, jobKey: String)(implicit ec: ExecutionContext): Future[Option[String]] = {
```

Comment:
`JOB_KEY` and `JOB_VALUE` seem to actually be named `STORE_KEY` and `STORE_VALUE` in the database. For consistency, I would changed the method name back, and then change the parameter name to `storeKey`.